### PR TITLE
Create test alisaGrinko

### PR DESCRIPTION
**Pull request by AlisaGrinko**
create test alisaGrinko

### CODE REVIEW

File: src/test/java/BraveQA.java [hunk 1]
```diff
@@ -82,4 +82,22 @@ public void Oxana_Wiki() throws InterruptedException {
...
+        WebElement downloadTab = browser.findElement(By.xpath("//a[.='Downloads']"));
+        downloadTab.click();
+
+        WebElement linkJavaAPI = browser.findElement(By.xpath("//a[contains(@href, '://seleniumhq.github.io/selenium/docs/api/java/index.html')]"));
+        linkJavaAPI.click();
+
+        WebElement framesTab = browser.findElement(By.xpath("//a[.='Frames']"));
+        Assert.assertEquals(framesTab.getText(), "FRAMES");
+
+        Thread.sleep(3000);
```
Comment (Stan575) [inline src/test/java/BraveQA.java:100]: It would be nice if you remove sleep to save time, and change line 86 accordingly

**Review by Stan575 — APPROVED** [review]
Good job on this test! Removing sleep before merging to main is up to you

File: src/test/java/BraveQA.java [hunk 2]
```diff
@@ -82,4 +82,22 @@ public void Oxana_Wiki() throws InterruptedException {
...
+    @Test
+    public void alisaGrinko() throws InterruptedException {
+
+        WebDriver browser = getDriver();
+        browser.get("https://www.selenium.dev/");
+
+        WebElement downloadTab = browser.findElement(By.xpath("//a[.='Downloads']"));
+        downloadTab.click();
+
+        WebElement linkJavaAPI = browser.findElement(By.xpath("//a[contains(@href, '://seleniumhq.github.io/selenium/docs/api/java/index.html')]"));
```
Comment (sergeonevdah) [inline src/test/java/BraveQA.java:94]: Change, please xPath on line 94

**Review by sergeonevdah — CHANGES_REQUESTED** [review]
Change please line 94, xPath

**Review by sergeonevdah — APPROVED** [review]
Good job 👍

**Pull request merged**